From 820cedceb44d9f8a6f26a22b495f14987f5610ee Mon Sep 17 00:00:00 2001
From: Xulin Sun <xulin.sun@windriver.com>
Date: Sun, 9 Oct 2016 15:30:07 +0800
Subject: [PATCH 755/756] can: flexcan: add missing renamed base to regs

Reference the commit 89af8746dea9(can: flexcan: use pointer to struct regs
instead of void pointer for mmio address space), the "base" of struct
flexcan_priv has been renamed to "regs", here add the missed ones.

Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/net/can/flexcan.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/net/can/flexcan.c b/drivers/net/can/flexcan.c
index 3100091..67cf745 100644
--- a/drivers/net/can/flexcan.c
+++ b/drivers/net/can/flexcan.c
@@ -682,7 +682,7 @@ static void flexcan_read_fifo(const struct net_device *dev,
 			      struct can_frame *cf)
 {
 	const struct flexcan_priv *priv = netdev_priv(dev);
-	struct flexcan_regs __iomem *regs = priv->base;
+	struct flexcan_regs __iomem *regs = priv->regs;
 	struct flexcan_mb __iomem *mb = &regs->mb[0];
 
 	flexcan_read_can_frame(priv, mb, cf);
@@ -695,7 +695,7 @@ static void flexcan_read_msg_buf(const struct net_device *dev,
 				 struct can_frame *cf, u32 msg_buf)
 {
 	const struct flexcan_priv *priv = netdev_priv(dev);
-	struct flexcan_regs __iomem *regs = priv->base;
+	struct flexcan_regs __iomem *regs = priv->regs;
 	struct flexcan_mb __iomem *mb = &regs->mb[msg_buf];
 
 	flexcan_read_can_frame(priv, mb, cf);
-- 
2.0.2

